### DISCUSSION TITLE
Give me Ideas

### DISCUSSION THREAD

**Discussion by ThatOneSillyGuyDon** [DEMOLITIONDON96/Demolition-Engine#104] [Ideas (Other)]
What do you want me to add in the engine (apart from all the fixes that are currently being worked on) for the release of 0.2.4?

###### (yes, some features from Theoyeah Engine are coming to this engine soon)

**Comment by ErenKaya31** [comment]
add sex (stage editor, note select in options)

**Comment by Theoyeah** [comment]
![image](https://user-images.githubusercontent.com/97792861/169645706-07a195d2-f652-486f-adb7-8b39d3ef405e.png)
heeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeee

**Comment by ghost** [comment]
Me must use WiggleEffect.

**Reply by ThatOneSillyGuyDon** [reply]
> Me must use WiggleEffect.

wiggle effect is already used on Thorns for the Dad Sprites

**Reply by ghost** [reply]
> > Me must use WiggleEffect.
> 
> wiggle effect is already used on Thorns for the Dad Sprites

I meant for the notes.
